### PR TITLE
Remove vlenb register

### PR DESCRIPTION
**Pull request by rez5427**
Sorry about the messy commit

### CODE REVIEW

**Comment by rez5427** [comment]
fix #565

File: model/riscv_sys_control.sail [hunk 1]
```diff
@@ -470,10 +470,7 @@ function init_sys() -> unit = {
   menvcfg.bits = zeros();
   senvcfg.bits = zeros();
   /* initialize vector csrs */
-  vlenb              = to_bits(xlen, 2 ^ (get_vlen_pow() - 3)); /* vlenb holds the constant value VLEN/8 */
-  /* VLEN value needs to be manually changed currently.
-   * See riscv_vlen.sail for details.
-   */
+
```
Comment (arichardson) [inline model/riscv_sys_control.sail:473]: Nit: drop this empty line, the comment refers to these CSRs

**Review by arichardson — APPROVED** [review]
History needs squashing change looks sensible.

**Review by jordancarlin — APPROVED** [review]
Might be nice to define vlen as `unsigned(get_vlenb()) * 8` since that is how vlenb is used so often, but that can be a later PR. This looks like a nice clean up as is.

**Comment by rez5427** [comment]
fix #574

**Comment by rez5427** [comment]
> Might be nice to define vlen as `unsigned(get_vlenb()) * 8` since that is how vlenb is used so often, but that can be a later PR. This looks like a nice clean up as is.

like this?

File: model/riscv_insts_vext_mask.sail [hunk 2]
```diff
@@ -31,7 +31,7 @@ mapping clause encdec = MMTYPE(funct6, vs2, vs1, vd) if extensionEnabled(Ext_V)
 function clause execute(MMTYPE(funct6, vs2, vs1, vd)) = {
   let SEW      = get_sew();
   let LMUL_pow = get_lmul_pow();
-  let num_elem = unsigned(vlenb) * 8;
+  let num_elem = unsigned(get_vlenb()) * 8;
```
Comment (Timmmm) [inline model/riscv_insts_vext_mask.sail:34]: You should be able to replace all these with `VLEN` too I think?

**Review by Timmmm — APPROVED** [review]
Nice improvement, LGTM!

**Pull request merged**